docs/_includes/js/overview.html

```diff
@@ -76,11 +76,9 @@ <h2 id="js-version-nums">Version numbers</h2>
 $.fn.tooltip.Constructor.VERSION // => "{{ site.current_version }}"
 {% endhighlight %}
 
-  <h2 id="js-disabled">No special fallbacks when JavaScript is disabled</h2>
+  <h2 id="js-disabled" data-toc-text="When JavaScript is disabled">No special fallbacks when JavaScript is disabled</h2>
   <p>Bootstrap's plugins don't fall back particularly gracefully when JavaScript is disabled. If you care about the user experience in this case, use <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/noscript"><code>&lt;noscript&gt;</code></a> to explain the situation (and how to re-enable JavaScript) to your users, and/or add your own custom fallbacks.</p>
 
-  <div class="bs-callout bs-callout-warning" id="callout-third-party-libs">
-    <h4>Third-party libraries</h4>
-    <p><strong>Bootstrap does not officially support third-party JavaScript libraries</strong> like Prototype or jQuery UI. Despite <code>.noConflict</code> and namespaced events, there may be compatibility problems that you need to fix on your own.</p>
-  </div>
+  <h2 id="callout-third-party-libs">Third-party libraries</h2>
+  <p><strong>Bootstrap does not officially support third-party JavaScript libraries</strong> like Prototype or jQuery UI. Despite <code>.noConflict</code> and namespaced events, there may be compatibility problems that you need to fix on your own.</p>
```

Comment:
This is one of the couple of places I intentionally changed to make the heading the same as the others on the page, so that it would show up in the sidebar.

**Before**

![screen shot 2016-02-08 at 4 19 35 am](https://cloud.githubusercontent.com/assets/86842/12881865/32a17996-ce1b-11e5-8649-94a5ea86ddf7.png)

**After**

![screen shot 2016-02-08 at 4 20 14 am](https://cloud.githubusercontent.com/assets/86842/12881874/4816de6a-ce1b-11e5-9de4-ff80153e515a.png)

There are other "warning" sections on the page that aren't displayed as callouts (such as "no special fallbacks" info above), so I went the easy route rather than trying to support the existing markup. Let me know if this is a deal-breaker.